torch/nn/quantized/modules/normalization.py

```diff
@@ -153,3 +153,279 @@ def from_float(cls, mod):
             mod.num_groups, mod.num_channels, mod.weight, mod.bias, float(scale), int(zero_point),
             mod.eps, mod.affine)
         return new_mod
+
+class InstanceNorm1d(torch.nn.InstanceNorm1d):
+    r"""Applies Instance Normalization over a 3D input (a mini-batch of 1D
```

Comment:
are these copied from floating point version? should we link to the fp version instead of copying?

Reply:
to clarify, you mean the docblock?

Reply:
Yeah I mean the docs